tests/eas/capacity_capping.py

```diff
@@ -83,12 +84,33 @@ class CapacityCappingTest(unittest.TestCase):
 
     """
 
+    critical_tasks = {
+        'linux': ['init', 'systemd', 'sh', 'ssh'],
+        'android': [
+            'sh', 'adbd',
+            'usb', 'transport',
+            # We don't actually need this task but on Google Pixel it apparently
+            # cannot be frozen, so the cgroup state gets stuck in FREEZING if we
+            # try to freeze it.
+            'thermal-engine'
+        ]
+    }
+    """
+    Dictionary mapping OS name to list of task names that we can't afford to
+    freeze when using freeeze_userspace.
+    """
```

Comment:
Can we import this instead of duplicating it? Since this is a bit of a hack anyway, we could just import the `Executor` class and then use `Executor.critical_tasks`.